crates/biome_js_formatter/src/utils/jsx.rs

```diff
@@ -542,6 +546,20 @@ impl<I: Iterator> JsxChildrenIterator<I> {
             })
             .as_ref()
     }
+
+    pub fn peek_next_next(&mut self) -> Option<&I::Item> {
```

Comment:
I have one question. Should `JsxChildrenIterator` be modified so that it can retrieve a specified number of elements ahead, like [`look_ahead`](https://github.com/rust-lang/rust/blob/a04d56b36d8f634abd7bdd64dd859a30655f1818/compiler/rustc_parse/src/parser/mod.rs#L1061)? I often commit to rustc, so I mentioned rustc as an example.

Reply:
i think following the implementation of `look_ahead` in rustc is good.

Reply:
Ok. I will implement it in a new pull request!